packages/react/src/components/SpinButton/SpinButton.styles.ts

```diff
@@ -192,49 +191,34 @@ export const getStyles = (props: ISpinButtonStyleProps): ISpinButtonStyles => {
         boxSizing: 'border-box',
         height: DEFAULT_HEIGHT,
         minWidth: DEFAULT_MIN_WIDTH,
-        selectors: {
-          // setting border using pseudo-element here in order to prevent:
-          // input and chevron buttons to overlap border under certain resolutions
-          ':after': {
-            pointerEvents: 'none',
-            content: "''",
-            position: 'absolute',
-            left: 0,
-            top: 0,
-            bottom: 0,
-            right: 0,
-            borderWidth: '1px',
-            borderStyle: 'solid',
-            borderColor: SpinButtonRootBorderColor,
-            borderRadius: effects.roundedCorner2,
+        ...getInputFocusStyle(SpinButtonRootBorderColor, effects.roundedCorner2),
```

Comment:
You may not want to call a focus style method here as the component isn't focused.

Reply:
the focus is state based, not :focus based